### PR TITLE
Setup React project

### PR DESCRIPTION
**Pull request by TuyizeeAnastase**
**### What does this PR do?**
setting up react route for dummy route "/"

**### Description of tasks to be completed**
-create a react app` npm init`
-install all needed packeges
-create a dummy route

**### How can be manually tested?**
-clone this repo
-run `npm install` in command will install all packages
-run npm run start will automatically open server in your default browser

**### what are relavant trello story?**
https://trello.com/c/RmsEGBhd/25-setup-react-project

### CODE REVIEW

**Comment by Ally4** [comment]
Good job Anastase, it starts the server but I get no message!. and I think it might be better for us to use one branch!.

**Comment by TuyizeeAnastase** [comment]
yeah, we must use one branch, and we will do it because we have the feature

**Review by Ally4 — DISMISSED** [review]

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: 'babel-loader',
+      },
+    ],
+  },
+  plugins: [
+    new HtmlWebpackPlugin({ template: './src/index.html' }),
+  ],
+};
```
Comment (Ntwali-Josue) [inline webpack.config.js:21]: Add single empty line on your files which has this git warning⛔
image

**Review by Ntwali-Josue — DISMISSED** [review]

File: webpack.config.js (hunk 1 above)
Comment (TuyizeeAnastase) [reply webpack.config.js:21]: Done.

**Review by Mugema123 — APPROVED** [review]

**Review by Ally4 — APPROVED** [review]

**Review by Ntwali-Josue — APPROVED** [review]

**Review by ceelogre — APPROVED** [review]

**Pull request merged**